cmd/zar/README.md

```diff
@@ -162,7 +162,7 @@ zar ls -l
 You will see all the indexes left behind. They are just zng files.
 If you want to see one, just look at it with zq, e.g.
 ```
-zq -t $ZAR_ROOT/20180324/1521912990.158766.zng.zar/microindex-type-ip.zng
+find $ZAR_ROOT -name 'microindex-type-ip.zng' | head -n 1 | xargs zq -t -
```

Comment:
No need for quotes.
```suggestion
find $ZAR_ROOT -name microindex-type-ip.zng | head -n 1 | xargs zq -t -
```

Also, what's going on might be a little clearer without xargs.
```sh
zq -t $(find $ZAR_ROOT -name microindex-type-ip.zng | head -n 1)
```